Sources/SmileID/Classes/SmileID.swift

```diff
@@ -90,13 +112,70 @@ public class SmileID {
         }
     }
 
-    public class func cleanup(jobIds: String...) throws {
-        try cleanup(jobIds: jobIds)
-    }
+    /// Submits a previously captured job to SmileID for processing.
+    ///
+    ///- Parameters:
+    ///   - jobId: The unique identifier for the job to be submitted.
+    public class func submitJob(
+        jobId: String,
+        deleteFilesOnSuccess: Bool
+    ) throws {
+        let jobIds = LocalStorage.getSubmittedJobs()
+        if !jobIds.contains(jobId) {
+            throw SmileIDError.invalidJobId
+        }
 
-    /// deletes the job ids list, whether in unsubmitted or submitted state
-    public class func cleanup(jobIds: [String]) throws {
-        try LocalStorage.delete(at: jobIds)
+        Task {
+            let zip: Data
+            do {
+                let authRequestFile = try? LocalStorage.fetchAuthenticationRequestFile(jobId: jobId)
+                let authRequest = AuthenticationRequest(
+                    jobType: authRequestFile!.jobType,
+                    enrollment: authRequestFile!.enrollment,
+                    jobId: authRequestFile!.jobId,
+                    userId: authRequestFile!.userId
+                )
+                let authResponse = try await SmileID.api.authenticate(request: authRequest).async()
+                let prepUploadFile = try? LocalStorage.fetchPrepUploadFile(jobId: jobId)
+                let prepUploadRequest = PrepUploadRequest(
+                    partnerParams: authResponse.partnerParams.copy(extras: prepUploadFile!.partnerParams.extras),
+                    allowNewEnroll: String(prepUploadFile!.allowNewEnroll), // TODO - Fix when Michael changes this to boolean
+                    timestamp: authResponse.timestamp,
+                    signature: authResponse.signature
+                )
+                let prepUploadResponse = try await SmileID.api.prepUpload(request: prepUploadRequest).async()
+                var allFiles = [URL]()
+                let selfieFileUrl = try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.selfie)
+                let frontDocumentUrl = try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.documentFront)
+                allFiles.append(contentsOf: [selfieFileUrl, frontDocumentUrl])
+                let backDocumentUrl = try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.documentBack)
+                allFiles.append(backDocumentUrl)
+                let livenessFiles = try LocalStorage.getFilesByType(jobId: jobId, fileType: FileType.liveness)
+                allFiles.append(contentsOf: livenessFiles)
+                let info = try LocalStorage.getInfoJsonFile(jobId: jobId)
+                allFiles.append(info)
```

Comment:
thoughts on doing it like this instead? this allows allFiles to be a `let` instead of `var`
```suggestion
                let allFiles = try LocalStorage.getFilesByType(jobId: jobId, fileType: FileType.liveness) + [
                    try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.selfie),
                    try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.documentFront),
                    try LocalStorage.getFileByType(jobId: jobId, fileType: FileType.documentBack),
                    try LocalStorage.getInfoJsonFile(jobId: jobId) 
                ].compactMap { $0 } // Filter out nil values
```

Reply:
bump